config/OS6_config_example.h

```diff
@@ -0,0 +1,392 @@
+#define MOTION_TIMEOUT 60 * 15 * 1000
+#define IDLE_OFF_TIME 5 * 60 * 1000
+#define FETT263_MULTI_PHASE
+#define FETT263_CLASH_STRENGTH_SOUND
+#define FETT263_DUAL_MODE_SOUND
+#define FETT263_MAX_CLASH 16
+#define FETT263_BM_CLASH_DETECT 7
+#define FETT263_SAY_BATTERY_PERCENT
+#define FETT263_SAY_COLOR_LIST
+#define FETT263_SAY_COLOR_LIST_CC
+//#define FETT263_SAVE_CHOREOGRAPHY
+#endif
+
+#ifdef CONFIG_PROP
+#include "../props/saber_fett263_buttons.h"
+#endif
+
+#ifdef CONFIG_PRESETS
+
+Preset presets[] = {
+
+{ "Fallen;common", "Fallen/tracks/JFO6.wav",
+/* copyright Fett263 FallenOrder (Primary Blade) OS6 Style
+https://fett263.s3.us-east-2.amazonaws.com/fett263-proffieOS6-style-library.html#FallenOrder
+OS6.5 v1.0
+This Style Contains 648 Unique Combinations
+Style Options:
+Default (0): Fallen Order Cal Kestis
+1: Fallen Order Second Sister
+
+--Effects Included--
+Preon Effect Options: Default (0): Disable, 1: Overload (PreOn Color), 2: Sparking (PreOn Color)
+Ignition Effect Options: Default (0): Standard Ignition, 1: SparkTip Ignition, 2: Dual Mode Wipe (Up = Fast)
+PowerUp Effect Options: Default (0): Disable, 1: Unstable Power Up Forward, 2: Power Burst Forward
+Retraction Effect Options: Default (0): Standard Retraction, 1: Dual Mode Wipe In (Up = Fast)
+CoolDown Effect Options: Default (0): Disable, 1: Unstable Cool Down Reverse, 2: Power Burst Reverse
+Lockup Effect: Intensity Lockup V1
+LightningBlock Effect: Responsive Lightning Block
+Drag Effect: Intensity Sparking Drag
+Melt Effect: Intensity Melt
+Blast Effect (Randomly Selected): Blast Wave (Random), Blast Ripple Fade, Blast Wave (Sound Based), Blast Ripple, Blast Fade (Sound Based)
+Clash Effect: Real Clash V1
+Swing Effect Options: Default (0): Disable, 1: Force Slow
+Battery Level: Blade Location (Green to Red)
+*/
+StylePtr<Layers<
+  ColorSelect<IntArg<STYLE_OPTION_ARG,0>,TrInstant,StripesX<Int<6000>,Scale<SlowNoise<Int<2000>>,Int<-1600>,Int<-3200>>,RgbArg<BASE_COLOR_ARG,Rgb<255,0,0>>,RandomPerLEDFlicker<Mix<Int<10280>,Black,RgbArg<BASE_COLOR_ARG,Rgb<255,0,0>>>,Mix<Int<1285>,Black,RgbArg<BASE_COLOR_ARG,Rgb<255,0,0>>>>,BrownNoiseFlicker<Mix<Int<1285>,Black,RgbArg<BASE_COLOR_ARG,Rgb<255,0,0>>>,Mix<Int<16448>,Black,RgbArg<BASE_COLOR_ARG,Rgb<255,0,0>>>,300>,RgbArg<BASE_COLOR_ARG,Rgb<255,0,0>>,RandomPerLEDFlicker<Black,Mix<Int<16448>,Black,RgbArg<BASE_COLOR_ARG,Rgb<255,0,0>>>>,RgbArg<BASE_COLOR_ARG,Rgb<255,0,0>>>,StripesX<Int<1500>,Scale<SlowNoise<Int<2500>>,Int<-3000>,Int<-5000>>,RgbArg<BASE_COLOR_ARG,Rgb<255,0,0>>,Mix<Int<10280>,Black,RgbArg<BASE_COLOR_ARG,Rgb<255,0,0>>>,RgbArg<BASE_COLOR_ARG,Rgb<255,0,0>>,Mix<Int<2570>,Black,RgbArg<BASE_COLOR_ARG,Rgb<255,0,0>>>,RgbArg<BASE_COLOR_ARG,Rgb<255,0,0>>>>,
+  ColorSelect<IntArg<SWING_OPTION_ARG,0>,TrInstant,AlphaL<Black,Int<0>>,
+  TransitionEffectL<TrConcat<TrExtend<30000,TrFade<300>>,Pulsing<RgbArg<SWING_COLOR_ARG,Rgb<255,255,255>>,Mix<Int<16384>,Black,RgbArg<SWING_COLOR_ARG,Rgb<255,255,255>>>,3000>,TrFade<300>>,EFFECT_FORCE>>,
```

Comment:
I think people are going to have a hard time seeing where the style ends if you have blank lines inside the styles.

Reply:
Or is the blank styles just github being weird?

Reply:
Is there any indentation here, I can't tell...

Reply:
I don't see blanks and it's indented on my end, maybe github is acting up?

Reply:
looks ok on my end.